test_rclcpp/test/test_services.py.in

```diff
@@ -0,0 +1,28 @@
+from launch import LaunchDescriptor
+from launch.exit_handler import primary_exit_handler
+from launch.launcher import DefaultLauncher
+
+
+def test_services():
+    ld = LaunchDescriptor()
+
+    ld.add_process(
+        cmd=['@TEST_SERVICES_SERVER_EXECUTABLE@'],
+        name='test_services_server',
+    )
+
+    ld.add_process(
+        cmd=['@TEST_SERVICES_CLIENT_EXECUTABLE@', 'test_services_server'],
+        name='test_services_client',
+        exit_handler=primary_exit_handler,
+    )
+
+    launcher = DefaultLauncher()
+    launcher.add_launch_descriptor(ld)
+    rc = launcher.launch()
+
+    assert rc == 0, 'The client did not receive any replies'
```

Comment:
This might not be the only reason why `rc` might be not zero. Maybe a phrase like for the other tests (https://github.com/ros2/system_tests/blob/ce48c4e0f7291e5a8c2ca37658b602457016f094/test_communication/test/test_publisher_subscriber.py.in#L24-L26) might be better.